python/paddle/optimizer/adam.py

```diff
@@ -30,6 +33,21 @@
 )
 from .optimizer import Optimizer
 
+if TYPE_CHECKING:
+    from typing_extensions import NotRequired
+
+    from paddle import Tensor
+    from paddle.nn.clip import GradientClipBase
+    from paddle.regularizer import WeightDecayRegularizer
+
+    from .lr import LRScheduler
+    from .optimizer import ParameterConfig
+
+    class AdamParameterConfig(ParameterConfig):
```

Comment:
```suggestion
    class _AdamParameterConfig(ParameterConfig):
```

非暴露，加个下划线吧，可以不用放到 `TYPE_CHECKING` 下面

Reply:
我把 ParameterConfig 定义在 optimizer.py 的 `TYPE_CHECKING` 下面（里面用到很多type_checking里导入的类），这里不放在 `TYPE_CHECKING` 下面是不是会找不到 ParameterConfig 啊

Reply:
:joy: 肯定是要改一起改啊